2018/5xxx/CVE-2018-5560.json

```diff
@@ -1,18 +1,101 @@
+      "ID": "CVE-2018-5560",
+      "STATE": "PUBLIC",
+      "TITLE": "Guardzilla All-In-One Video Security System Hard-Coded Credential"
    },
-   "data_format" : "MITRE",
-   "data_type" : "CVE",
-   "data_version" : "4.0",
-   "description" : {
-      "description_data" : [
+   "affects": {
+      "vendor": {
+         "vendor_data": [
+            {
+               "product": {
+                  "product_data": [
+                     {
+                        "product_name": "Guardzilla All-In-One Video Security System",
+                        "version": {
+                           "version_data": [
+                              {
+                                 "affected": "<=",
+                                 "version_name": "2018",
+                                 "version_value": "2018"
+                              }
+                           ]
+                        }
+                     }
+                  ]
+               },
+               "vendor_name": "Practecol, LLC"
+            }
+         ]
+      }
+   },
+   "credit": [
+      {
+         "lang": "eng",
+         "value": "Discovered by Nick McClendon, Andrew Mirghassemi, Charles Dardaman, INIT_6, and Chris."
+      }
+   ],
+   "data_format": "MITRE",
+   "data_type": "CVE",
+   "data_version": "4.0",
+   "description": {
+      "description_data": [
          {
-            "lang" : "eng",
-            "value" : "** RESERVED ** This candidate has been reserved by an organization or individual that will use it when announcing a new security problem.  When the candidate has been publicized, the details for this candidate will be provided."
+            "lang": "eng",
+            "value": "A reliance on a static, hard-coded credential in the design of the cloud-based storage system of Practecol's Guardzilla All-In-One Video Security System allows an attacker to view the private data of all users of the Guardzilla device. As of the initial publishing of this CVE, this issue remains unresolved and unacknowledged by the vendor.\nThis issue affects:\nPractecol, LLC Guardzilla All-In-One Video Security System"
          }
       ]
-   }
+   },
+   "impact": {
+      "cvss": {
+         "attackComplexity": "LOW",
+         "attackVector": "NETWORK",
+         "availabilityImpact": "NONE",
+         "baseScore": 8.6,
+         "baseSeverity": "HIGH",
+         "confidentialityImpact": "HIGH",
+         "integrityImpact": "NONE",
+         "privilegesRequired": "NONE",
+         "scope": "CHANGED",
+         "userInteraction": "NONE",
+         "vectorString": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:C/C:H/I:N/A:N",
+         "version": "3.0"
+      }
+   },
+   "problemtype": {
+      "problemtype_data": [
+         {
+            "description": [
+               {
+                  "lang": "eng",
+                  "value": "CWE-798: Use of Hard-coded Credentials"
+               }
+            ]
+         }
+      ]
+   },
+   "references": {
```

Comment:
refsource should be MISC for both of these. CONFIRM is just for response from vendor confirming the vuln. Also need to add attributes per MITRE's previous guidance: title (plaintext short description of resource) and name (the URL again, due to how these get processed by MITRE IIRC). Example with this format: https://github.com/CVEProject/cvelist/pull/1486/files

Reply:
Done!

Reply:
Note that the lack of a CONFIRM does throw a warning in the [Vulnogram](https://vulnogram.github.io/), but the vendor didn't actually confirm the vuln so...

Reply:
Done!